src/Command/GenerateJsonCommand.php

```diff
@@ -158,27 +158,79 @@ private function generatePrestaShopModulesJson(
      */
     private function addVersionsUnderDelopment(array $prestashopVersions): array
     {
-        // Add current development version so that its module's list is available, but only if it is not part of the released ones.
-        $developmentVersion = '9.0.0';
-        $nextPatchVersion = '8.1.4';
+        if (empty($prestashopVersions)) {
```

Comment:
If latest version is `8.1.4`, it will generate `8.1.5`, `8.2.0` and `9.0.0`.
In the future, if latest version is `9.0.0`, it will generate `8.1.5`, `8.2.0`,`9.0.1`,`9.1.0` and ,`10.0.0`.
In the future, if latest version is `9.1.4`, it will generate `8.1.5`, `8.2.0`,`9.1.5`,`9.2.0` and ,`10.0.0`.

Is it OK?